examples/01-basic/11-placeholder/App.tsx

```diff
@@ -0,0 +1,26 @@
+import "@blocknote/core/fonts/inter.css";
+import { BlockNoteView } from "@blocknote/mantine";
+import "@blocknote/mantine/style.css";
+import { useCreateBlockNote } from "@blocknote/react";
+import { locales } from "@blocknote/core";
+
+export default function App() {
+  // Set the user locale.
+  const userLocale = "en";
+
+  const placeholder = "This is a custom placeholder";
+
+  // Creates a new editor instance.
+  const editor = useCreateBlockNote({
+    dictionary: {
+      ...locales[userLocale],
+      placeholders: {
```

Comment:
Besides the default, please also show how you can change the "heading" placeholder

Reply:
add a comment to explain code
```
// We override the `placeholders` in our dictionary
```